src/main/java/invtweaks/InvTweaksItemTree.java

```diff
@@ -48,8 +55,13 @@
     private String rootCategory;
     @NotNull
     private List<OreDictInfo> oresRegistered = new ArrayList<>();
+    
+    @NotNull
+    private List<ItemStack> allGameItems = new ArrayList<ItemStack>();
```

Comment:
Nothing outside of here seems to ever use this, is there a reason for it to exist? It's potentially a good chunk of memory in highly-modded installs just sitting around persistently.